test/unit/deadline_client/cli/test_cli_asset.py

```diff
@@ -42,6 +52,45 @@ def upload_group_mock(asset_group_mock):
     )
 
 
+@pytest.fixture
+def mock_asset_manifest():
+    return AssetManifest(paths=[], hash_alg=HashAlgorithm("xxh128"), total_size=0)
+
+
+@pytest.fixture
+def mock_attachment_settings():
+    return Attachments(manifests=[], fileSystem="").to_dict
+
+
+@pytest.fixture
+def mock_init_objects():
+    with patch.object(S3AssetManager, "__init__", lambda self, *args, **kwargs: None), patch.object(
+        S3AssetUploader, "__init__", lambda self, *args, **kwargs: None
+    ), patch.object(JobAttachmentS3Settings, "__init__", lambda self, *args, **kwargs: None):
+        yield
+
+
+@pytest.fixture
+def mock_update_manifest():
+    with patch.object(asset_group, "update_manifest", return_value=mock_asset_manifest) as mock:
```

Comment:
```suggestion
def mock_update_manifest(mock_asset_manifest):
    with patch.object(asset_group, "update_manifest", return_value=mock_asset_manifest) as mock:
```

We need to take in the fixture here in order to use the return value. I believe this code is setting the `return_value` as the fixture function (Callable), not its return value (AssetManifest).